src/main/java/com/targomo/client/api/request/config/multigraph/SourceParameter.java

```diff
@@ -0,0 +1,49 @@
+package com.targomo.client.api.request.config.multigraph;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.targomo.client.Constants;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SourceParameter {
```

Comment:
some javadoc would be nice I guess, no idea what this is/can be used for

Reply:
done.